client/src/Pages/CreateItinerary.jsx

```diff
@@ -173,7 +174,7 @@ const CreateItinerary = () => {
 
                         <div className="field_info_container">
                             <label htmlFor="country">Country</label>
-                            <input type="text" placeholder={"Enter country"} value={country} onChange={(e) => setCountry(e.target.value)} />
+                            <CountrySelect onChange={(e) => setCountry(e.value)}/>
```

Comment:
i think the country select is a good addition but the weird ui part where you can see the red start date and end date text is a lil weird so maybe can just stick to text input?